src/zksync-specifics/examples/general-paymaster.md

```diff
@@ -0,0 +1,151 @@
+## Using the zkUsePaymaster Cheatcode in General Flow Paymaster Contracts
+
+This example covers the use of a general flow paymaster contract. 
```

Comment:
```suggestion
This example covers the use of a general flow for a paymaster contract. 
```

Reply:
The "General flow" is how the docs describe it